Masonry/MASViewAttribute.h

```diff
@@ -14,6 +14,8 @@
  */
 @interface MASViewAttribute : NSObject
 
+NS_ASSUME_NONNULL_BEGIN
+
 /**
  *  The view which the reciever relates to. Can be nil if item is not a view.
  */
```

Comment:
ok, fixed it. fixed also the designated initializer which allows `view` to be `nil` too. did not fix the convenience initializer, as `view` ist used for both `view` and `item` there, and `item` may _not_ be `nil`.